### PR TITLE
Add verified environment information.

### PR DESCRIPTION
**Pull request by blackpiglet**
Signed-off-by: Xun Jiang <jxun@vmware.com>

### CODE REVIEW

**Review by euclidsun — DISMISSED** [review]
looks good to me! thanks!

File: README.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ For all other storage systems, please check with your storage vendor.  If your s
 durable snapshot storage you may want to consider 
 [Velero's Restic Integration](https://velero.io/docs/latest/restic)
 
+## Verified environment
+Velero CSI plugin v0.3 is tested with Velero v1.9 on the following environments:
+* EKS: ebs.csi.aws.com driver is tested with helm chart version helm-chart-aws-ebs-csi-driver-2.6.8 and external-snapshotter version is v5.0.1. v6.0.1 external-snapshotter is also tested, but aws-ebs-csi-driver doesn't have a official helm chart tag supporting v6.0.1 external-snapshotter yet.
```
Comment (reasonerjt) [inline README.md:23]: There is a tag for aws-ebs-driver that pins to v6.0.1:
https://github.com/kubernetes-sigs/aws-ebs-csi-driver/releases/tag/helm-chart-aws-ebs-csi-driver-2.6.11

And I think this is what we used after v0.3.0-rc1 to verify the functionalities.

Comment (blackpiglet) [reply README.md:23]: Got it. Thanks.

**Review by ywk253100 — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -18,6 +18,12 @@ For all other storage systems, please check with your storage vendor.  If your s
 durable snapshot storage you may want to consider 
 [Velero's Restic Integration](https://velero.io/docs/latest/restic)
 
+## Verified environment
+Velero CSI plugin v0.3 is tested with Velero v1.9 on the following environments:
+* EKS: ebs.csi.aws.com driver is tested with helm chart version helm-chart-aws-ebs-csi-driver-2.6.8 and external-snapshotter version is v5.0.1. v6.0.1 external-snapshotter is also tested with EBS CSI driver helm chart version helm-chart-aws-ebs-csi-driver-2.6.11.
+* AKS: disk.csi.azure.com driver is tested. CSI related controllers are installed in control-plan, so cannot find the exactly version. Tested AKS versions are 1.23.3 and v1.22.6.
```
Comment (reasonerjt) [inline README.md:24]: Sorry just saw another trivial issue:
It should be `on control plane`, not `in control-plan`

Comment (blackpiglet) [reply README.md]: Modified.

**Review by reasonerjt — APPROVED** [review]
lgtm

**Review by ywk253100 — APPROVED** [review]

**Pull request merged**